extra/alacritty-msg.man

```diff
@@ -2,28 +2,27 @@
 .SH NAME
 alacritty-msg \- Send messages to Alacritty
 .SH "SYNOPSIS"
-alacritty msg [OPTIONS] [MESSAGES]
+alacritty msg [OPTIONS] [MESSAGE] [MESSAGE_OPTIONS]
```

Comment:
```suggestion
alacritty msg [OPTIONS] <MESSAGE> [MESSAGE_OPTIONS]
```

This has absolutely nothing to do with your changes, but thinking about the usage of `alacritty msg` I think `[MESSAGE]` should be `<MESSAGE>` instead. Since it's not optional; we always want *some* message sent.